modules/javafx.swt/src/main/java/javafx/embed/swt/FXCanvas.java

```diff
@@ -184,6 +184,16 @@ private double getScaleFactor() {
             } catch (Exception e) {
                 // FAIL silently should the reflection fail
             }
+        } else if (SWT.getPlatform().equals("win32")) {
+            if(swtDPIUtilMethod == null) {
+        	return 1.0;
```

Comment:
There is a TAB character on this line. Please convert it to spaces (to avoid failure in checkWhiteSpace and jcheck).

[MINOR] since you need to fix up the TAB anyway, can you add a space after the `if` in `if(swtDPIUtilMethod == null)` on the preceding line?